pkg/exchange/binance/exchange.go

```diff
@@ -538,6 +538,8 @@ func (e *Exchange) submitMarginOrder(ctx context.Context, order types.SubmitOrde
 	if len(order.TimeInForce) > 0 {
 		// TODO: check the TimeInForce value
 		req.TimeInForce(binance.TimeInForceType(order.TimeInForce))
+	} else {
+		req.TimeInForce(binance.TimeInForceTypeGTC)
```

Comment:
TimeInForce is only valid for limit order, you might need to check the order type.

If you set timeInForce for market order, the market order submission will fail.

Reply:
Thanks, you are right. Fixed.